code/modules/antagonists/wizard/equipment/chuunibyou_spell.dm

```diff
@@ -28,12 +28,7 @@
 	if(ishuman(cast_on))
 		var/mob/living/carbon/human/human_cast_on = cast_on
 		human_cast_on.dropItemToGround(human_cast_on.glasses)
-		var/obj/item/clothing/head/wizard/wizhat = human_cast_on.head
-		if(istype(wizhat))
-			to_chat(human_cast_on, span_notice("Your [wizhat] transforms into an eyepatch."))
-			qdel(wizhat)
-		else
-			to_chat(human_cast_on, span_notice("An eyepatch pops into existence over one of your eyes."))
```

Comment:
Please see the modularization guidelines at https://github.com/Monkestation/Monkestation2.0/pull/219/files for how to modularize correctly